affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedTestConfiguration.kt

```diff
@@ -3,32 +3,14 @@ package com.dropbox.affectedmoduledetector
 /**
  * Used to configure which variant to run for affected tasks by adding following block to modules
  * affectedTestConfiguration{
- *  variantToTest = "debug"
- *  jvmTestBackup = "test"
+ *  assembleAndroidTestTask = "assembleDevDebugAndroidTest"
  * }
  */
 open class AffectedTestConfiguration {
 
-    /**
-     *  Sets variant for all affected tasks
-     *  By default `run` tasks will use debug as the variant to test ie
-     *  gradlew runAffectedUnitTests will run testDebugUnitTest
-     *
-     */
-    var variantToTest: String? = null
-        get() {
-            return field ?: "debug"
-        }
-
-    /**
-     * when [jvmTest] task is not found we will try to run [jvmTestBackup]
-     * this is normally used for modules that are not android variant aware
-     */
-    var jvmTestBackup = "test"
-
-    val assembleAndroidTestTask get() = "assemble${variantToTest?.capitalize()}AndroidTest"
-    val runAndroidTestTask get() = "connected${variantToTest?.capitalize()}AndroidTest"
-    val jvmTest get() = "test${variantToTest?.capitalize()}UnitTest"
+    var assembleAndroidTestTask : String? = "assembleDebugAndroidTest"
```

Comment:
Opening these up to be nullable.  If they are set to null, we'll ignore that module when creating the main task.